src/components/portrait/portrait.component.tsx

```diff
@@ -129,60 +177,4 @@ const Portrait = ({
   return renderComponent();
 };
 
-Portrait.propTypes = {
-  ...marginPropTypes,
-  /** The size of the Portrait. */
-  size: PropTypes.oneOf(["XS", "S", "M", "ML", "L", "XL", "XXL"]),
-  /** A custom image URL. */
-  src: (props) => {
```

Comment:
Should we have an invariant for the `src` prop? As I don't think any consuming JS projects won't get any feedback that they shouldn't set both `src` and `gravatar` props at the same time

Reply:
I thought about it and decided that it's not worth it in this case. It's not breaking anything crucial and even if it does it is an immediately visible visual change.

Reply:
That's a good point I hadn't checked if it breaks when both are applied, the fact it doesn't means we shouldn't add an invariant 👍